docs/developer/architecture/rollout-disruption-budget.md

```
@@ -0,0 +1,24 @@
+# Disruption Budget
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)